datasets/youtube.py

```diff
@@ -1,21 +1,59 @@
 import os
 import json
 import argparse
+import glob
 
-parser = argparse.ArgumentParser()
-parser.add_argument('--input-path', '-i', required = True)
-parser.add_argument('--output-path', '-o', required = True)
-parser.add_argument('--strip', nargs = '*', default = ['begin', 'end'])
-args = parser.parse_args()
 
-transcript = []
-for info_name in os.listdir(args.input_path):
-	if not info_name.endswith('.json'):
-		continue
-	
-	info_path = os.path.join(args.input_path, info_name)
-	transcript.extend(dict(audio_path = info_path.replace('.json', ''), **{k : v for k, v in t.items() if k not in args.strip} ) for t in json.load(open(info_path)).get('transcript', []))
+def main(args):
+	transcripts = []
+	for i, info_path in enumerate(glob.glob(os.path.join(args.input_path, f'*/*/*.json'))):
```

Comment:
what's the point of '*/*/*.json'? Does it mean that the existing [`scripts/youtube.sh`](https://github.com/vadimkantorov/convasr/blob/master/scripts/youtube.sh#L25) code is now invalid?